appsec-kit-backend/src/main/java/com/vaadin/appsec/backend/AppSecService.java

```diff
@@ -164,6 +164,8 @@ public void scheduleAutomaticScan() {
                 initialDelay = secondsUntilNextScan;
             }
         }
+        LOGGER.debug("Scheduling automatic scan every " + autoScanPeriod
+                + " seconds");
```

Comment:
This is the right place and time to log this task. Could we remove the other similar log entry [here](https://github.com/vaadin/appsec-kit/blob/fix/notification-access/appsec-kit-v8/src/main/java/com/vaadin/appsec/v8/service/AppSecServiceInitListener.java#L35) and [here](https://github.com/vaadin/appsec-kit/blob/fix/notification-access/appsec-kit-v7/src/main/java/com/vaadin/appsec/v7/service/AppSecKitInitializer.java#L57)? I think these are not logged in the right time because of the vulnerability scan takes longer. Also, we don't need two log entries for this.